generators/app/templates/src/StarterKit/Startup/Startup.cs

```diff
@@ -81,11 +88,24 @@ public virtual void ConfigureServices(IServiceCollection services)
       #region Logging
 
       services.AddLoggingEngine();
+      services.AddTransient<LoggingHandler>();
 
       #endregion
 
       //--dataaccess-startupServices--
 
+      services.AddServiceAgents(settings =>
```

Comment:
Ik vermoed dat de ServiceAgents standaard niet werden ingeladen en dat dat door deze aanpassing wel gebeurt.

@erikseynaeve: Weet jij of het de bedoeling is om de ServiceAgents op deze manier in te laden? Ik ben niet in detail op de hoogte van de ontwikkelingen aan de toolboxen.

Reply:
Dit zou beter weggelaten worden. Als dit nodig was voor testen, zou dit via een unit- of integratietest kunnen gebeuren. Nu gaat dit bij elke projectgeneratie meegenomen worden.
Het is de bedoeling ook om de ServiceAgents-toolbox obsolete te maken. Er komt geen vervanging. I.p.v. zou er bv. wel een base-class + een verzameling veelgebruikte delegating handlers mee opgenomen worden in de starterkit. Dit wordt normaal mee opgenomen bij de verdere ontwikkeling voor de toolboxen.